index.html

```diff
@@ -10,37 +10,40 @@
 <body>
     <div class="container">
         <div class="todo">
-            <header class="header">
+            <header class="todo__header">
                 <h1>Todos</h1>
             </header>
 
-            <input type="text" name="new-todo" class="new-todo" placeholder="What needs to be done?">
-            <div class="checkbox-field checkbox-field--small">
-                <label class="checkbox-field__label">
-                    <input type="checkbox" class="checkbox-field__input">
-                    Mark all as complete
-                </label>
-            </div>
+            <div class="todo__content">
+                <input type="text" name="todo__new-todo-field" class="todo__new-todo-field" placeholder="What needs to be done?">
+
+                <div class="todo__complete-all-field checkbox-field checkbox-field--small">
```

Comment:
Wanted to introduce this possibly strange class names to discuss this scenario: How do you handle using a Block declared elsewhere as an Element of another Block in another instance. (eg. element _`todo__complete-all-field`_ is element of block _`todo`_ but suppose it to be a block defined elsewhere (eg. block _`checkbox-field`_). Can you use both classnames in this case (ie classname for being an element, and classname for being a block)? Or would you prefer handling this scenario with Sass mixins?